dmutils/file.py

```diff
@@ -0,0 +1,47 @@
+import os
+import boto3
+from werkzeug.utils import secure_filename
+from flask import current_app
+from io import BytesIO
+
+
+def allowed_file(filename):
+    return '.' in filename and \
+        filename.rsplit('.', 1)[1] in current_app.config.get('ALLOWED_EXTENSIONS')
+
```

Comment:
This checks the filename only which doesn't necessarily match the file contents.. Worth checking mime type of file?